src/main/java/org/broadinstitute/consent/http/models/DarCollection.java

```diff
@@ -121,27 +120,11 @@ public void setUpdateUserId(Integer updateUserId) {
   }
 
   public Map<String, DataAccessRequest> getDars() {
-    if (Objects.isNull(dars)) {
-      return new HashMap<>();
-    }
     return dars;
   }
 
-  public void setDars(Map<String, DataAccessRequest> dars) {
-    this.dars = dars;
-  }
-
   public void addDar(DataAccessRequest dar) {
-    if (Objects.isNull(dars)) {
-      this.setDars(new HashMap<>());
-    }
-    if (Objects.nonNull(dar)) {
-      String referenceId = dar.getReferenceId();
-      DataAccessRequest savedDar = dars.get(referenceId);
-      if (Objects.isNull(savedDar)) {
-        dars.put(referenceId, dar);
-      }
-    }
+    dars.putIfAbsent(dar.getReferenceId(), dar);
```

Comment:
Why not just put?  Couldn't the value of `dar` have changed?

Reply:
> Why not just put? Couldn't the value of dar have changed?

The short answer is, this is what it was doing before, so this change preserves the current behavior.

Looking at its usage, it's only used while processing the result of a SQL operation, in `DarCollectionReducer`. In that case, I wouldn't expect to see it used more then once for the same dar. So `put()` _should_ be safe.